### DISCUSSION TITLE
aliyundrive-webdav服务端日志有可以显示webdav客户端访问的阿里云盘具体文件（访问的文件名，访问时间，访问者ip等）吗？

### DISCUSSION THREAD

**Discussion by untitled99** [messense/aliyundrive-webdav#436] [Q&A] [answered]
如题。

主要是想查看nplayer播放器所播放的阿里云视频文件的记录详情。

记得以前在某个服务器部署出错的时候，显示了具体的上传的文件名，如下：
`
2022-04-24T11:50:26.081172Z ERROR aliyundrive_webdav::vfs: upload file chunk 1 failed file_id=62653964a72591f04dc843babe2b0029b185e686 file_name=happykids_20220424-1945.mp4 size=18
3811124 error=error sending request for url (https://bj29.cn-beijing.data.alicloudccp.com/...?partNumber=1&uploadId=...&x-oss-access-key-id=...&x-oss-expires=1650804596&x-oss-signature=...BiBnM%3D&x-oss-signature-version=OSS2): operation timed out
`

也就是说，可以打开更详细的日志显示功能？具体我应该在哪里打开？谢谢。

**Comment by messense** [comment, marked as answer]
`--debug`
